.kokoro/presubmit_tests_functions.cfg

```diff
@@ -0,0 +1,15 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Download secrets from Cloud Storage.
+gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/python-docs-samples"
+
+# Tell the trampoline which build file to use.
+env_vars: {
+    key: "TRAMPOLINE_BUILD_FILE"
+    value: "github/python-docs-samples/.kokoro/system_tests.sh"
+}
+
+env_vars: {
+    key: "NOX_SESSION"
+    value: "functions and py27 and not venv"
```

Comment:
I'm not familiar with this file, what does it do? Esp. this part

Reply:
This file tells Kokoro what tests should be run. The `NOX_SESSION` variable is in _Pytest-style_ [key format](http://nox.readthedocs.io/en/latest/usage.html#specifying-one-or-more-sessions).